### DISCUSSION TITLE
CAN'T INSTALL BRAVE, VLC PLAYER ETC on MINT. UNUSUALLY LARGE FILE SIZE IS SHOWN

### DISCUSSION THREAD

**Discussion by ukmint7** [linuxmint/discussions#516] [Questions]
How to install Brave or VLC Player on mint. Software Manager shows extraordinarily large file sizes instead of original file sizes, and fails to install. What to do. Even tried reinstalling Mint but of no use. Please help.

**Comment by spamegg1** [comment]
That's how Flatpak works, it's very space hungry.

You can prefer DEB packages instead:

`apt install vlc` should work for VLC.

For Brave, you can follow these instructions https://brave.com/linux/#debian-ubuntu-mint
